roles/set_scale/meta/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+dependencies:
+  - role: check_deps
+  - role: ensure_machineset
```

Comment:
hum, I never linked roles this way, I included them explicitly in the `main.yml`
could you add a comment at the beginning of `roles/set_scale/tasks/main.yml ` to mention that this role will be executed before `roles/set_scale/tasks/main.yml ` starts?